databricks/koalas/spark/accessors.py

```diff
@@ -851,6 +851,27 @@ def apply(self, func, index_col: Optional[Union[str, List[str]]] = None) -> "ks.
             )
         return output.to_koalas(index_col)
 
+    def repartition(
+        self, numPartitions: int, index_col: Optional[Union[str, List[str]]] = None
```

Comment:
`num_partitions` instead of `numPartitions`?

Reply:
I think this should work without `index_col`, returns the same DataFrame except for partitioning.